README.md

```diff
@@ -28,6 +28,35 @@ $ pug-lint [options] <file ...>
 * `-c, --config <path>`: [configuration file](#configuration-file) path
 * `-r, --reporter <reporter>`: error reporter; console - default, inline
 
+#### Lint multiples files in npm script
+
+First install [glob-exec](https://www.npmjs.com/package/glob-exec)
+
+```shell
+npm i -D glob-exec
+```
+
+```json
+{
+  "scripts": {
+    "lint:pug": "glob-exec \"./src/**/*.pug\" -- \"pug-lint {{files.join(' ')}}\""
```

Comment:
Is the extra dependency `glob-exec` really needed? Could you achieve the same, but simpler, with standard `xargs`?

Even better, just `pug-lint $(git ls-files '*.pug')`?

More generally, I'm not sure such instructions add real value to the README: they're really generic.

Reply:
Thanks for suggestion. I change script to `git ls-files 'src/**/*.pug' | xargs pug-lint`